website/docs/commands/publishing/publish-local.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Publish Local
+sidebar_position: 21
+---
+
+import {ChainedSnippets} from "../../../src/components/MarkdownComponents.js";
+
+The `publish local` sub-command publishes a Scala CLI project in the local Ivy2
+repository, just like how `sbt publishLocal` or `mill __.publishLocal` do. This
+repository usually lives under `~/.ivy2/local`, and is taken into account most of
+the time by most Scala tools when fetching artifacts.
+
+To publish locally a Scala CLI project, you can run
```

Comment:
What options are required to publish project locally? I guess, we need to have at least organization and name provided and I think the doc should mention it.

Reply:
Should be fixed too.